doc/cephadm/install.rst

```diff
@@ -399,38 +356,74 @@ Different deployment scenarios
+To configure a Ceph cluster to run on a single host, use the 
+``--single-host-defaults`` flag when bootstrapping. For use cases 
+of this, see :ref:`one-node-cluster`.
 
-The ``--single-host-defaults`` flag sets the following configuration options::
+The ``--single-host-defaults`` flag sets the following configuration 
+options::
 
   global/osd_crush_choose_leaf_type = 0
   global/osd_pool_default_size = 2
   mgr/mgr_standby_modules = False
    
-For more information on these options, see :ref:`one-node-cluster` and ``mgr_standby_modules`` in :ref:`mgr-administrator-guide`.
+For more information on these options, see :ref:`one-node-cluster` 
+and ``mgr_standby_modules`` in :ref:`mgr-administrator-guide`.
+
+.. _cephadm-airgap:
 
 Deployment in an isolated environment
 -------------------------------------
 
-You can install Cephadm in an isolated environment by using a custom container registry. You can either configure Podman or Docker to use an insecure registry, or make the registry secure. Ensure your container image is inside the registry and that you have access to all hosts you wish to add to the cluster.
+You can install Cephadm in an isolated environment by using a custom 
+container registry. Sometimes also referred as airgap. 
+You can either configure Podman or Docker to use 
+an insecure registry, or make the registry secure. Ensure your container 
+image is inside the registry and that you have access to all hosts 
+you wish to add to the cluster.
 
 Run a local container registry:
 
 .. prompt:: bash #
 
    podman run --privileged -d --name registry -p 5000:5000 -v /var/lib/registry:/var/lib/registry --restart=always registry:2
 
-If you are using an insecure registry, configure Podman or Docker with the hostname and port where the registry is running.
+If you are using an insecure registry, configure Podman or Docker 
+with the hostname and port where the registry is running.
+
+.. note:: 
+
+  For every host which accesses the local insecure registry, you will 
+  need to repeat this step on the host.
 
-.. note:: For every host which accesses the local insecure registry, you will need to repeat this step on the host.
+Next, push your container image to your local registry:
 
-Next, push your container image to your local registry.
+* Ceph container iamge. See :ref:`containers`.
+* Prometheus container image
+* Node exporter container image
+* Grafana container image
+* Alertmanager container image
+
+Now, create a temporary configuration file for setting the montoring
+images. (See :ref:`cephadm_monitoring-images`)::
+
+      $ cat <<EOF > initial-ceph.conf
+      [mgr]
+      mgr/cephadm/container_image_prometheus *<hostname>*:5000/prometheus
+      mgr/cephadm/container_image_node_exporter *<hostname>*:5000/node_exporter
+      mgr/cephadm/container_image_grafana *<hostname>*:5000/grafana
+      mgr/cephadm/container_image_alertmanager *<hostname>*:5000/alertmanger
```

Comment:
The other challenging part is to find which containers and which versions.
Since this is not documented newcomers have a hard time finding this information.
If the only source to this is the cephadm code, maybe add that.